guide/contribute/content-guidelines.md

```diff
@@ -93,3 +93,7 @@ The guide is a work in progress and needs to evolve to stay relevant. What we wr
 ### Write in the open
 
 It's a good idea to ask for directions by seeking feedback early. This may stop you from wandering too far in the wrong direction. There is a whole community around you that's ready to jump in and help.
+
+---
+
+Moving on to the next section, let’s take a look at the [Illustration guidelines]({{ '/guide/contribute/illustration-guidelines/' | relative_url }}).
```

Comment:
```suggestion
Now that we've covered general content guidelines, let’s take a closer look at [Illustration guidelines]({{ '/guide/contribute/illustration-guidelines/' | relative_url }}).
```

Reply:
I see your rocket emoji. Does that mean you approve of this change? Want to make sure you're OK with it before merging.

Reply:
I still need to learn to communicate clearly, right? LOL

Lets do this. I like your version better.